### ISSUE TITLE
No connectors working

### ISSUE THREAD

**Issue by RaisinBran36** [manga-download/hakuneko#4883]
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

Upon opening the app and refreshing the connector I'm using (Mangakakalot) I get 11 errors and 5 warnings in the network section.
When trying to download a manga, it immediately fails with another error.

I tried on a previous day with multiple other connectors and the issue remained.  This time I did an un/re-install, but no dice.

The one time I used the app before was on a strict network connection with tons of blocked websites and yet I was able to download around 50 chapters from various series. Now, I get this error on public McDonalds wifi and my phone's wifi hotspot.

[
![Untitled](https://user-images.githubusercontent.com/39650205/182730111-005a7928-bd87-4d75-b8a4-4e00c6310904.png)
![Untitled1](https://user-images.githubusercontent.com/39650205/182730118-177a7b91-28a0-4cef-925c-84f29b23e414.png)
](url)

 - Version [6.1.7] on Windows 64-bit

**Comment by Sheepux** [comment]
try "hakuneko nightly"

**Comment by Kingjon96** [comment]
I was having this same issue and I also tried on nightly after it was fixed and it still gives the same 403 error.

**Comment by Sheepux** [comment]
Welll, this report is about "no connector working", does is in fact mean only Mangakakalot ?

**Comment by RaisinBran36** [comment]
> Welll, this report is about "no connector working", does is in fact mean only Mangakakalot ?

As I said previously, “ I tried on a previous day with multiple other connectors and the issue remained. This time I did an un/re-install, but no dice.”

I did more troubleshooting and resolved the issue by deleting the local files. 
it turns out that doing a simple uninstall will make a future reinstall not work.
The files from %appdata% need to be deleted as well.
